bitnami/grafana-operator/templates/grafana.yaml

```
@@ -132,6 +132,9 @@ spec:
             - name: grafana
               image: {{ include "common.images.image" (dict "imageRoot" .Values.grafana.image "global" .Values.global) }}
               imagePullPolicy: {{ .Values.grafana.image.pullPolicy }}
+              {{- if .Values.grafana.extraEnvVars }}
+              env: {{- toYaml .Values.grafana.extraEnvVars | nindent 16 }}
+              {{- end }}
```

Please use the same format than the example I mentioned before.

```suggestion
              {{- if .Values.grafana.extraEnvVars }}
              env: 
                {{- include "common.tplvalues.render" (dict "value" .Values.grafana.extraEnvVars "context" $) | nindent 16 }}
              {{- end }}
```